CONTRIBUTING.md

```diff
@@ -0,0 +1,113 @@
+# Contributing to Revolver
+
+## Structure
+Revolver works with a pluggable interface for `drivers` and `plugins` that are executed per cloud account.
+* `drivers` implement cloud API calls, both query and update.
+* `plugins` implement power cycling logic.
```

Comment:
Technically power-cycling is only 1-2 plugins, plus tag validation. May be extended by other plugins.